About.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="stylesheet" href="./sass/main.css">
+</head>
+
+<body>
+
+    <header>
+        <div class="pre-navbar bg-color">
+            <ul>
+                <li><img src="./img/Social1.png" alt=""></li>
+                <li><img src="./img/Social2.png" alt=""></li>
+                <li>SignIn</li>
+                <li>My Page</li>
+                <li>Language</li>
+            </ul>
+        </div>
+        <nav class="nav-bar">
+            <!-- Mobile-nav -->
+            <div class="mobile-nav">
+                <input id="menu-toggle" type="checkbox" />
+                <label class='menu-button menu-button-container' id="menuButton" for="menu-toggle">
+                </label>
+
+                <ul class="menu">
+                    <li><a class="menu-item " href="./index.html">Home</a></li>
+                    <li><a class="menu-item" href="">BuyNow</a></li>
+                    <li><a class="menu-item" href="">Contact</a></li>
+                </ul>
+            </div>
```

Comment:
- [ ] it will be nice if on the desktop version for the about page, you have the home link instead of the about link. Kindly fie it
![Capture](https://user-images.githubusercontent.com/85095702/169387314-c99f2b03-5aa7-4bb9-a0d6-27c8999ec0ca.PNG)